lms/templates/admin/assignment/show.html.jinja2

```diff
@@ -19,14 +19,5 @@
       <legend class="label has-text-centered">Course</legend>
       {{ macros.course_preview(request, assignment.course) }}
     </fieldset>
-
-    <fieldset class="box mt-6">
```

Comment:
I added the roster information to the admin pages. 

Removing it now, it's easier to revert the commit rather than adapt it to the new changes on the service.

As we are starting to expose this information on the dashboard that's easier to support rather than do both admin pages and dashboards.